utils/helpers.tsx

```diff
@@ -0,0 +1,52 @@
+import { BLOCKS, MARKS } from '@contentful/rich-text-types';
+import { documentToHtmlString } from '@contentful/rich-text-html-renderer';
+
+export const formatDate = (dateString: string) => {
```

Comment:
This a common problem to solve.

Maybe using a well-known library like [date-fns](https://date-fns.org/) to format dates could be simpler. 😉